src/bundle/Resources/public/js/scripts/fieldType/ezobjectrelationlist.js

```diff
@@ -215,7 +215,15 @@
             return [...relationsContainer.querySelectorAll('[type="checkbox"]')];
         };
         const attachRowsEventHandlers = () => {
-            findOrderInputs().forEach((item) => item.addEventListener('blur', updateSelectedItemsOrder, false));
+            findOrderInputs().forEach((item) => {
+                item.addEventListener('blur', updateSelectedItemsOrder, false);
+                item.addEventListener('change', focusOnElement, false);
```

Comment:
If this is only on FF I don't see a point to attach this event in all browsers. Please add a check if it is FF and attach it only in this case.

Reply:
the added condition only for FF attach